src/Exceptionless.Core/Pipeline/030_CheckForRegressionAction.cs

```diff
@@ -52,7 +52,16 @@ public override async Task ProcessBatchAsync(ICollection<EventContext> contexts)
                     {
                         var version = _semanticVersionParser.Parse(versionGroup.Key, versionCache) ?? _semanticVersionParser.Default;
                         if (version < fixedInVersion)
+                        {
+                            foreach (var ctx in stackGroup.Where(s => s.Organization.HasPremiumFeatures))
```

Comment:
I thought about adding a new pipeline action, but it would basically be a copy of this pipeline action with the same version parsing.